tests/testthat/test-mrkdown_labels.R

```diff
@@ -0,0 +1,153 @@
+test_that("markdown column labels - no spanning",{
+
+  mock_data <- tibble::tibble(rowlbl1 =c(rep("Completion Status",12),rep("Primary reason for withdrawal",28)),
```

Comment:
```suggestion
  mock_data <- tibble(rowlbl1 =c(rep("Completion Status",12),rep("Primary reason for withdrawal",28)),
```

Reply:
I copied this from quite a few of the other tests. Should I remove it from those?